LICENSE

```diff
@@ -963,3 +963,33 @@ necessary.  Here is a sample; alter the names:
 
 That's all there is to it!
 
+------------------------------------------------------------------------------
```

Comment:
Does this interfere with the way GH detects licenses?

I looked at Apache Torque (which also bundles JGit) and it looks like they use separate license files + a NOTICE file. 
http://www.apache.org/dyn/closer.cgi/db/torque/torque-4.0/binaries/torque-generator-4.0-bin.tar.gz

Reply:
There is no problem here. There are several other open-source dependencies that we handle the same way.